### ISSUE TITLE
Labels not associated with inputs

### ISSUE THREAD

**Issue by jvine** [sul-dlss/earthworks#642]
On the Web services page: https://earthworks.stanford.edu/catalog/stanford-pz657cj8819/web_services

Not clear to me why these are presented as inputs when there is no input allowed. In any case, the  inputs with druid values have labels nearby but they're not associated by `for="id"` or nesting. 

<img width="544" alt="Screen Shot 2020-06-12 at 1 17 57 PM" src="https://user-images.githubusercontent.com/6945691/84542865-26e4bb00-acaf-11ea-9ec7-053f72858a07.png">
